pkg/config/kind.go

```diff
@@ -85,14 +85,17 @@ func (ks KindSet) Set(k Kind) KindSet {
 	return ks | (1 << k)
 }
 
+// make gas happy
+func ignoreErrors(args ...interface{}) {}
+
 func (ks KindSet) String() string {
 	buf := pool.GetBuffer()
 	defer pool.PutBuffer(buf) // fine to pay the defer overhead; this is out of request path
 
-	fmt.Fprint(buf, "[")
+	ignoreErrors(fmt.Fprint(buf, "["))
```

Comment:
Are u using different linter versions? 'cause why is this surfacing now?

Also, just using _, _ = fmt.Fprintf is not sufficient?